examples/decoupledibpm/oscillatingcylinder2dRe100_GPU/README.md

```diff
@@ -1,12 +1,22 @@
 # 2D flow around inline oscillating cylinder (Re=100)
 
-Run the example using 4 CPU processes and 1 GPU:
+If `Makefile` is present in this folder, you can compile the example with
```

Comment:
I suggest we can add something like:

(Makefile of this example only appears when using `make copy-examples` directly under build folder. If using `make copy-examples EXAMPLES_DIR=<...>` to copy examples to other locations (e.g., installation folder), there will be no Makefile.)